library/globals.inc.php

```diff
@@ -174,13 +168,6 @@
       xl('Choose your default encounter view')
     ),
 
-    'css_header' => array(
-      xl('Theme'),
-      'css',
-      'style_light.css',
-      xl('Pick a CSS theme.')
-    ),
-
     'gbl_nav_area_width' => array(
       xl('Navigation Area Width'),
```

Comment:
Add a menu_styling_vertical (Vertical Menu Style) global which does:
0 for Tree
1 for Sliding